src/db/mysql/models.rs

```diff
@@ -317,7 +317,9 @@ impl MysqlDb {
             .execute(&self.conn)?;
             collections::table.select(last_insert_id).first(&self.conn)
         })?;
-        self.coll_cache.put(id, name.to_owned())?;
+        if !self.session.borrow().in_transaction {
```

Comment:
Let's get rid of this cache put entirely, since this is a write operation (`create_collection`)